### PR TITLE
[PE-1] Add lerna to project

### PR DESCRIPTION
**Pull request by aleksanderfret**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/alemikolo/plan-event.git"
+  },
+  "bugs": {
+    "url": "https://github.com/alemikolo/plan-event/issues"
+  },
+  "scripts": {
+    "lerna-c": "lerna create",
+    "lerna-b": "lerna bootstrap"
```
Comment (marta-fret) [inline package.json]: I wonder if these names will be easy to use in the console

File: packages/server/package.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
+{
+  "name": "plan-event-server",
+  "description": "Plan An Event Server",
```
Comment (marta-fret) [inline packages/server/package.json]: Maybe: Plan An Event - server part ?

File: packages/web/package.json [hunk 3]
```diff
@@ -0,0 +1,20 @@
+{
+  "name": "plan-event-web",
```
Comment (marta-fret) [inline packages/web/package.json]: Maybe plan-event-browser or plan-event-client?

**Review by marta-fret — APPROVED** [review]
Just sharing thoughts about names :-)

**Comment by aleksanderfret** [comment]
@all-contributors please add @marta-fret for review

**Comment by aleksanderfret** [comment]
@all-contributors please add @alemikolo for code

**Comment by aleksanderfret** [comment]
@all-contributors please add @alemikolo for ideas

**Pull request merged**